### PR TITLE
Update version of icons

### PR DESCRIPTION
**Pull request by ashton-vaz**
https://www.npmjs.com/package/@salesforce-ux/icons

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,7 +129,7 @@
 		"@babel/preset-env": "^7.14.2",
 		"@babel/preset-react": "^7.12.7",
 		"@salesforce-ux/design-system": "^2.9.4",
-		"@salesforce-ux/icons": "9.x",
+		"@salesforce-ux/icons": "10.x",
```
Comment (ashton-vaz) [inline package.json:132]: https://www.npmjs.com/package/@salesforce-ux/icons

**Comment by interactivellama** [comment]
@ashton-vaz Please sign the CLA. Since these are icons. Please post a screen shot of the icons storybook page.

**Comment by interactivellama** [comment]
I tracked down changes from icons 9.x to 10.x. The repo is not public though, but in 10.0.0, we optimized some of the existing svg files. Here are the notes from that icons release:

```
SVG Optimizations
Removed Generator comments
Removed id attributes from path and g tags
Removed title tags (alternative text provided when icon is consumed by the design system)
Removed desc tags
Removed empty g tags
Removed defs tags
Removed xml tags
Removed version attribute from svg tag
Removed DOCTYPE declarations
Removed xmlns:xlink attribute from svg tag
Removed deprecated icon font
```

**Comment by ashton-vaz** [comment]
> I tracked down changes from icons 9.x to 10.x. The repo is not public though, but in 10.0.0, we optimized some of the existing svg files. Here are the notes from that icons release:
> 
> ```
> SVG Optimizations
> Removed Generator comments
> Removed id attributes from path and g tags
> Removed title tags (alternative text provided when icon is consumed by the design system)
> Removed desc tags
> Removed empty g tags
> Removed defs tags
> Removed xml tags
> Removed version attribute from svg tag
> Removed DOCTYPE declarations
> Removed xmlns:xlink attribute from svg tag
> Removed deprecated icon font
> ```

Does 10.x refer to only 10.0.0? In that case, should I change it to 10.2.2 to reflect the most up to date version? @interactivellama

**Comment by interactivellama** [comment]
Using "10.x" will use the latest version of the library. I'm just sharing what the breaking change is so you can validate that the upgrade works correctly.

**Comment by ashton-vaz** [comment]
> @ashton-vaz Please sign the CLA. Since these are icons. Please post a screen shot of the icons storybook page.

Screen Shot 2021-12-15 at 12 27 41 PM

**Comment by ashton-vaz** [comment]
@interactivellama I just signed the CLA, how do I re-trigger the check?

**Comment by interactivellama** [comment]
It looks like opening and closing the PR will work for the CLA... It's been flakey recently.

**Review by interactivellama — APPROVED** [review]

**Comment by interactivellama** [comment]
@ashton-vaz Please proceed with release PR.

**Comment by ashton-vaz** [comment]
> @ashton-vaz Please proceed with release PR.

Thank you!

**Pull request merged**